hypothesis-python/src/hypothesis/strategies.py

```diff
@@ -935,20 +939,28 @@ def characters(
 @cacheable
 @defines_strategy_with_reusable_values
 def text(
-    alphabet=None,  # type: Union[Sequence[Text], SearchStrategy[Text]]
+    alphabet=characters(
+        blacklist_categories=('Cs',)
+    ),  # type: Union[Sequence[Text], SearchStrategy[Text]]
     min_size=None,   # type: int
     average_size=None,   # type: int
     max_size=None  # type: int
 ):
     # type: (...) -> SearchStrategy[Text]
     """Generates values of a unicode text type (unicode on python 2, str on
-    python 3) with values drawn from alphabet, which should be an iterable of
-    length one strings or a strategy generating such. If it is None it will
-    default to generating the full unicode range (excluding surrogate
-    characters). If it is an empty collection this will only generate empty
-    strings.
+    python 3) with values drawn from alphabet, which should be a strategy for
+    length-one strings or a valid ``whitelist_characters`` argument for
+    ``characters()`` to create such a strategy.
+
+    The default alphabet strategy can generate the full unicode range but
+    excludes surrogate characters because they are invalid in the UTF-8
+    encoding.  You can use :func:`~hypothesis.strategies.characters` without
+    arguments to find surrogate-related bugs such as :bpo:`34454`.
 
     min_size and max_size have the usual interpretations.
+    Note that Python measures string length by counting codepoints: U+00C5
+    ``Å`` is a single character, while U+0041 U+030A ``Å`` is two - the ``A``,
+    and a combining ring above.  This strategy does not include normalisation.
```

Comment:
In the context of Unicode it might be better to use the more familiar US spelling, “normalization”.

Also, “does not include normalisation” doesn't really make clear what is or isn't being handled or generated by the strategy.